addon/config.js

```diff
@@ -7,6 +7,10 @@ export default Object.assign(
     authEndpoint: "/protocol/openid-connect/auth",
     tokenEndpoint: "/protocol/openid-connect/token",
     logoutEndpoint: "/protocol/openid-connect/logout",
+    userinfoEndpoint: "/protocol/openid-connect/userinfo",
```

Comment:
hmm thinking about it again, if someone doesn't configure those routes correctly the addon will fail. The specification doesn't define default routes so shoudln't we enforce that user of the addon needs to define those routes and if not added fail?
I guess this would be much safer. In the future this might be simplified with discovery url.

Reply:
I agree, removed all default values and throw an error if the endpoints are not defined